[2024-07-13T02:40:37.083+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T02:40:37.101+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.transform_data manual__2024-07-13T02:40:27.527530+00:00 [queued]>
[2024-07-13T02:40:37.107+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.transform_data manual__2024-07-13T02:40:27.527530+00:00 [queued]>
[2024-07-13T02:40:37.107+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-13T02:40:37.124+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-07-13 02:40:27.527530+00:00
[2024-07-13T02:40:37.132+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1066) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-13T02:40:37.134+0000] {standard_task_runner.py:63} INFO - Started process 1068 to run task
[2024-07-13T02:40:37.134+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***_book_api_pipeline', 'transform_data', 'manual__2024-07-13T02:40:27.527530+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/***_book_api_dags.py', '--cfg-path', '/tmp/tmpbt_o6aom']
[2024-07-13T02:40:37.135+0000] {standard_task_runner.py:91} INFO - Job 31: Subtask transform_data
[2024-07-13T02:40:37.177+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow_book_api_pipeline.transform_data manual__2024-07-13T02:40:27.527530+00:00 [running]> on host e6b306b9c204
[2024-07-13T02:40:37.264+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zainal' AIRFLOW_CTX_DAG_ID='***_book_api_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T02:40:27.527530+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-13T02:40:27.527530+00:00'
[2024-07-13T02:40:37.265+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T02:40:37.461+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-13T02:40:37.461+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T02:40:37.469+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***_book_api_pipeline, task_id=transform_data, run_id=manual__2024-07-13T02:40:27.527530+00:00, execution_date=20240713T024027, start_date=20240713T024037, end_date=20240713T024037
[2024-07-13T02:40:37.508+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-13T02:40:37.525+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-13T02:40:37.532+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
